rest/server_context.go

```diff
@@ -82,6 +83,27 @@ func NewServerContext(config *ServerConfig) *ServerContext {
 		sc.startStatsReporter()
 	}
 
+	if config.Replications != nil {
+		go func() {
+			time.Sleep(kServerStartupWait)
```

Comment:
Can we trigger the start of config-based replications after the REST endpoints are started (in RunServer) to address this, instead of using a timer?